.eleventy.js

```diff
@@ -27,10 +27,10 @@ const downloadImage = async path => {
 }
 
 const processImage = async img => {
-  let { distPath, assetPath } = config;
+  let { distPath, assetPath, attribute = 'src' } = config;
```

Comment:
For the sake of consistency with the other config variables, is there any chance you could move the default assignment to the config definition on line 7 please?

Reply:
Done 👍

Reply:
Legend. Thanks!